### ISSUE TITLE
The results in the paper were not reproduced, and I hope to receive your guidance

### ISSUE THREAD

**Issue by goodai2** [zzli2022/LANS#2]
I used the pre trained model provided in the PGPS model and trained it on LANS. The final test results did not meet the indicators in the paper. Here is the result of the final round of loss。
2024-10-24 01:26:01,002 - INFO: ----------Epoch:538, training loss is 0.0236 ---------
2024-10-24 01:26:04,306 - INFO: Epoch: [539][ 0/54]	Time 2.735 (2.735)	Data 0.566 (0.566)	Loss 1.6686e-02 (1.6686e-02)	lr: 0.000003
2024-10-24 01:26:40,245 - INFO: Epoch: [539][20/54]	Time 2.116 (1.842)	Data 0.000 (0.027)	Loss 2.3232e-02 (2.3756e-02)	lr: 0.000003
2024-10-24 01:27:18,218 - INFO: Epoch: [539][40/54]	Time 2.351 (1.869)	Data 0.000 (0.014)	Loss 2.5310e-02 (2.4058e-02)	lr: 0.000003
2024-10-24 01:27:42,415 - INFO: ----------Epoch:539, training loss is 0.0245 ---------
2024-10-24 01:27:42,964 - INFO: ------------------- Train Finished -------------------
The completion test result on PGPS9K is
2024-10-24 10:20:24,083 - INFO: ----------Epoch:530, test answer_acc 0.6287, equation_acc 0.5958 ---------
Can you help me take a look at this issue? Or can you provide a new pre trained model? Thank you very much!

**Comment by zzli2022** [comment]
The pre-training method of PGPSNet does not introduce layout information. You need to first perform pre-training that introduces layout information according to the method of this paper. We will provide a warehouse and weights for pre-training using layout information.

**Comment by goodai2** [comment]
> PGPSNet的预训练方法没有引入布局信息，需要先按照本文的方法进行引入布局信息的预训练，我们会提供使用布局信息进行预训练的仓库和权重。

Looking forward to your open source

**Comment by goodai2** [comment]
when you will provide a warehouse and weights for pre-training using layout information？

**Comment by zzli2022** [comment]
try to use the pretraining code in github. I will set a huggingface to upload weights

**Comment by goodai2** [comment]
I couldn't find the pre-training code you provided. Could you send me a link?

**Comment by zzli2022** [comment]
You can visit https://github.com/zzli2022/LANS/blob/main/LANS_Pretraining.zip

**Comment by goodai2** [comment]
Is the point_loc used in the pre-training the newly annotated content in the dataset? If so, how to get the newly annotated data?
